webui/react/src/pages/ModelDetails/ModelHeader.tsx

```diff
@@ -0,0 +1,137 @@
+import { LeftOutlined } from '@ant-design/icons';
+import { Breadcrumb, Button, Dropdown, Menu, Modal, Space } from 'antd';
+import React, { useCallback, useMemo } from 'react';
+
+import Icon from 'components/Icon';
+import InfoBox, { InfoRow } from 'components/InfoBox';
+import InlineEditor from 'components/InlineEditor';
+import Link from 'components/Link';
+import { relativeTimeRenderer, userRenderer } from 'components/Table';
+import TagList from 'components/TagList';
+import { useStore } from 'contexts/Store';
+import { paths } from 'routes/utils';
+import { ModelItem } from 'types';
+import { formatDatetime } from 'utils/date';
+
+import css from './ModelHeader.module.scss';
+
+interface Props {
+  model: ModelItem;
+  onDelete: () => void;
+  onSaveDescription: (editedDescription: string) => Promise<void>
+  onSaveName: (editedName: string) => Promise<void>;
+  onSwitchArchive: () => void;
+  onUpdateTags: (newTags: string[]) => Promise<void>;
+}
+
+const ModelHeader: React.FC<Props> = (
+  {
+    model, onDelete, onSwitchArchive,
+    onSaveDescription, onUpdateTags, onSaveName,
+  }: Props,
+) => {
+  const { auth: { user } } = useStore();
+
+  const infoRows: InfoRow[] = useMemo(() => {
+    return [ {
+      content:
+      (<Space>
+        {userRenderer(model.username, model, 0)}
```

Comment:
**suggestion:** the `<xxx>Renderer` functions are generally for Table use only. How about taking `Avatar` component and adding a prop like `showName?: boolean` or something like to support showing both the user bubble + the username/name.